installer_mac/make_installer.sh

```diff
@@ -76,17 +98,17 @@ rm "$RSRCS/BuildInfo.txt"
 
 # create distribution.xml
 
-if [[ -d $VST2 ]]; then
-	VST2_PKG_REF='<pkg-ref id="com.vemberaudio.vst.pkg"/>'
-	VST2_CHOICE='<line choice="com.vemberaudio.vst.pkg"/>'
-	VST2_CHOICE_DEF="<choice id=\"com.vemberaudio.vst.pkg\" visible=\"true\" start_selected=\"true\" title=\"VST\"><pkg-ref id=\"com.vemberaudio.vst.pkg\"/></choice><pkg-ref id=\"com.vemberaudio.vst.pkg\" version=\"${VERSION}\" onConclusion=\"none\">Surge_VST2.pkg</pkg-ref>"
+if [[ -d $PRODUCTS/$VST2 ]]; then
+	VST2_PKG_REF='<pkg-ref id="com.vemberaudio.vst2.pkg"/>'
```

Comment:
I think this is supposed to just be `vst`, which is the commonly referred to term for VST 2.4? It really doesn't matter much as long as the installer still says "VST" to the end user, which it appears to.

Reply:
It's also in some `Info.plist` somewhere; I'd just leave it as `vst`